site/docs/es/guide/introduction.md

```diff
@@ -156,5 +197,18 @@ Pulsa `Ctrl+C` en tu terminal para detener el proceso.
 Esto detendrá tu bot.
 Entonces, necesitas repetir los pasos 2 y 3.
 
-¿Estás listo?
+::: tip Espera, ¿qué?
+
+Instalar Node.js y configurar todo correctamente lleva mucho tiempo.
+Si nunca has hecho esto antes, deberías esperar encontrarte con un montón de problemas confusos que son difíciles de solucionar.
+
+Por eso esperamos que sepas cómo configurar tu sistema, o que seas capaz de aprender por ti mismo.
+(Instalar Node.js _de la forma correcta_ es tan complicado que no cabe en esta página).
+
+Si te sientes perdido en este punto, deberías dejar Node.js y usar [Deno](#prerequisites-for-deno) en su lugar.
+
+:::
+
+¿Aún confiado?
+¡Genial!
 ¡[Empieza](./getting-started.md#comenzando-en-nodejs)! :robot:
```

Comment:
@habemuscode This anchor link is not correct.
```suggestion
¡[Empieza](./getting-started.md#comenzando-en-node-js)! :robot:
```

Reply:
It’s correct, thanks!